src/screens/Profiles/ProfilePages/index.js

```diff
@@ -117,8 +119,14 @@ class ProfilePage extends PureComponent {
         id: userId,
         ...changes,
       }, { include: 'waifu' });
+
+      // Bust the cover cache
+      if (data && data.coverImage) {
+        const bustedCovers = this.applyCacheBust(data.coverImage);
+        data.coverImage = bustedCovers;
+      }
+
       this.setState({ profile: data });
-      await fetchCurrentUser();
```

Comment:
🤔 Will this cause out-of-sync issues if we don't re-fetch it here?
Say for example, creating a comment where we apply `.user = currentUser;`

Reply:
i don't think it will cause out-of-sync errors since the id in the data returned and the id of the current user will always be consistent.

i had a look at all the uses of `currentUser` in the file and we are always exclusively using `currentUser.id` fo everything.

Reply:
Yeah, was thinking about avatar changes in reference to how we apply user to comments before pushing them into the list data locally.